jobfunnel/glassdoor_base.py

```diff
@@ -0,0 +1,88 @@
+import re
+
```

Comment:
Can you please do 
`git mv jobfunnel/glassdoor.py jobfunnel/glassdoor_base.py` so that we don't lose the history for changes in this file?

I can't see what changes, if any, were made to the logic in this file which is from that original source.